src/main/cpp/t05_power.cpp

```diff
@@ -28,8 +28,17 @@
 using namespace std;
 
 //function power
-
+double power(double a, int n)
+{
+	if (n != 0)
+		return pow (a, n);
+	if (n == 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'n==0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512856)